src/main/pg/autopilot_wing.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef USE_WING
+
+#include <stdint.h>
+
+#include "pg/pg.h"
+
+typedef struct apConfig_s {
+} apConfig_t;
```

Comment:
Wondering if an empty structure might cause any issues?

Reply:
Good question. I looked it up and it is indeed undefined behavior. A pointer to an empty struct probably points to something else in the program with funny behavior "for free"

Reply:
yeah just for safety i can fill it with `uint8 dummy`.

Reply:
Yes it seems compiler dependent. See also https://gcc.gnu.org/onlinedocs/gcc-3.3.1/gcc/Empty-Structures.html#Empty%20Structures

@ledvinap what is your recommendation ?